news/2018-03-08-TWC-2018-registrations-open.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "osu!taiko World Cup 2018: Registrations now open!"
+date: 2018-03-08 13:00:00 +0000
+---
+ 
+Once more the drumfest of the osu!taiko World Cup is back for its 8th installment! Some minor tweaks to the ScoreV2 system alongside a fresh map selection team are our premises for an exciting tournament! The Registration Phase already started, no time to waste!
```

Comment:
alongside with
has already started, so there's no time to waste!

Reply:
No. "alongside a" is fine. 
"along with" is an alternative, "alongside with" doesn't exist.